### PR TITLE
Added documentation for SetupFlags

### PR DESCRIPTION
**Pull request by mxxo**
Here are some docs for `SetupFlags`. Can I ask for advice on wording for `SQ_AFF`? 

`SQPOLL` creates a kernel thread to poll the submission queue and `SQ_AFF` has something to do with that thread, but I don't entirely understand what. My best understanding is there's some thread in userland resposible for the `SubmissionQueue`, and `SQ_AFF` forces the kernel thread to be spawned by the CPU of the userland thread.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -63,9 +63,34 @@ pub use cqe::{CompletionQueue, CompletionQueueEvent};
 pub use registrar::Registrar;
 
 bitflags::bitflags! {
+    /// `IoUring` initialization flags for advanced use cases.
+    ///
+    /// ```
```
Comment (withoutboats) [inline src/lib.rs]: I'm getting permission denied when I run this test as my normal user (running Linux 5.2.11).

Maybe this test should be `no_run` if some of these configurations require special privileges?

**Comment by mxxo** [comment]
I think keeping the names the same is not a bad idea. Maybe it helps keep everyone on the same page for what the flags do. 

The `nix` package follows this matchy-matchy convention for their interface flags, see  
https://docs.rs/nix/0.15.0/nix/net/if_/struct.InterfaceFlags.html 
and the `SIOCGIFFLAGS` section here: 
http://man7.org/linux/man-pages/man7/netdevice.7.html

File: src/lib.rs (hunk 1 above)
Comment (mxxo) [reply src/lib.rs]: Right, thank you. Should be fixed by 43271ac

**Comment by withoutboats** [comment]
Thanks!

**Pull request merged**